alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultConfigurationAccessor.java

```diff
@@ -95,16 +95,14 @@ public DefaultConfigurationAccessor(final RegisteredDescriptorRepository registe
     @Override
     public List<ConfigurationJobModel> getAllJobs() {
         final List<ConfigGroupEntity> jobEntities = configGroupRepository.findAll();
-        final Map<UUID, Set<ConfigGroupEntity>> jobMap = new HashMap<>();
+        final SetMap<UUID, ConfigGroupEntity> jobMap = new SetMap(new HashMap<>());
         for (final ConfigGroupEntity entity : jobEntities) {
             final UUID entityJobId = entity.getJobId();
-            if (!jobMap.containsKey(entityJobId)) {
-                jobMap.put(entityJobId, new HashSet<>());
-            }
-            jobMap.get(entityJobId).add(entity);
+            jobMap.add(entityJobId, entity);
         }
 
-        return jobMap.entrySet()
+        return jobMap.getMap()
+                   .entrySet()
```

Comment:
We call .getMap().entrySet() multiple times. We should add a helper method to SetMap that gets this.

Reply:
If SetMap was a Map you would just call .entrySet().